### PR TITLE
Merge master into release-0.33.x

### PR DESCRIPTION
**Pull request by paulrosenzweig**

### CODE REVIEW

File: bin/version [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-VERSION="v0.33.0"
+VERSION="v0.34.0-snapshot"
```
Comment (mazameli) [inline bin/version]: Should this be `VERSION="v0.33.1"`?

File: docs/faq.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ To manually fix field or table names if they still look wrong, you can go to the
 
 ### Can I add headings, free text, section dividers, or images to my dashboards?
 
-Yes, by using [text cards](users-guide/07-dashboards.html#adding-headings-or-descriptions-with-text-cards).
+Yes, by using [text cards](users-guide/07-dashboards.md#adding-headings-or-descriptions-with-text-cards).
```
Comment (mazameli) [inline docs/faq.md]: This is wrong; it should be `.html` in cases where we're using anchor/hash links. By some odd quirk of a doc build process, these kinds of links don't properly get `.md` converted to `.html`, which caused a bunch of broken links I had to fix manually.

File: docs/users-guide/04-asking-questions.md [hunk 3]
```diff
@@ -24,7 +24,40 @@ To ask a question about a table of data, we usually do some **filtering** and/or
 
 Filtering just means narrowing things down based on certain criteria. You're probably already familiar with filtering when looking for something online, like when shopping. Maybe you only want to see olive-colored pants, or books where the author's last name is "Steinbeck," or pictures of people wearing olive-colored pants reading John Steinbeck.
 
-You can do the same kind of thing with data in Metabase. Just click the Filter button in the top-right of the screen to open the filter sidebar. You'll see a list of all of the columns in this table, as well as columns from tables that are related to the one you're looking at. Depending on the column you pick, you'll see slightly different options for your filter.
+### Using the Question Builder interface
```
Comment (mazameli) [inline docs/users-guide/04-asking-questions.md]: This is wrong. `release-0.33.x` had the correct content for this section:

```
You can do the same kind of thing with data in Metabase. Just click the Filter button in the top-right of the screen to open the filter sidebar. You'll see a list of all of the columns in this table, as well as columns from tables that are related to the one you're looking at. Depending on the column you pick, you'll see slightly different options for your filter.
```

**Review by mazameli — CHANGES_REQUESTED** [review]

File: bin/version (hunk 1 above)
Comment (paulrosenzweig) [reply bin/version]: yes!

File: docs/faq.md (hunk 2 above)
Comment (paulrosenzweig) [reply docs/faq.md]: This was a conflict that I resolved incorrectly. I'll fix it.

File: docs/users-guide/04-asking-questions.md (hunk 3 above)
Comment (paulrosenzweig) [reply docs/users-guide/04-asking-questions.md]: I don't remember seeing this as a conflict, so I'm a bit confused what went wrong. I'm fixing this one, but let me know if there are other weird errors and I can dig more.

**Review by mazameli — APPROVED** [review]

**Pull request merged**